header1.h

```diff
@@ -0,0 +1,90 @@
+
+using namespace std;
+
+enum governmentType
```

Comment:
Enumerators should be named like constant. For exemple kDemocracy, kRepublic, ...